src/table/table.scss

```diff
@@ -122,265 +249,341 @@
+      border-bottom: 0;
     }
+    // #endregion Selection
 
-    &.iui-tables-row-new {
+    // #region Statuses & states
+    &.iui-new {
       font-weight: $iui-font-weight-semibold;
-
-      > .iui-tables-filename {
-        position: relative;
-      }
-
-      > .iui-tables-filename::before {
+      
+      > .iui-main-column::before {
         @include themed {
           color: t(iui-color-foreground-positive);
         }
         content: '•';
         position: absolute;
         left: 0;
         font-size: $iui-font-size-title;
-      }
+      }   
     }
 
-    &.iui-tables-row-disabled {
+    &.iui-disabled,
+    &.iui-disabled + .iui-expanded-content {
+      font-style: italic;
+      cursor: not-allowed;
       @include themed {
         color: t(iui-text-color-muted);
       }
-      font-style: italic;
-      cursor: not-allowed;
 
-      &:hover {
+      &:hover > .iui-slot > .iui-more-options {
+        visibility: hidden;
+      }
+
+      img,
+      svg:not(.iui-radial),
+      .iui-user-icon {
+        filter: $iui-icons-color-multicolor-disabled;
+      }
+    }
+
+    &.iui-positive {
+      &,
+      + .iui-expanded-content {
         @include themed {
-          background-color: t(iui-color-background-1);
+          box-shadow: inset $iui-xxs 0 0 0 t(iui-icons-color-positive);
         }
 
-        > .iui-tables-slot > .iui-tables-menu {
-          visibility: hidden;
+        *::selection {
+          @include themed {
+            background-color: rgba(
+              t(iui-color-foreground-positive-rgb),
+              t(iui-opacity-4)
+            );
+          }
         }
       }
 
-      > .iui-tables-cell {
-        > img,
-        > svg,
-        > .iui-user-icon,
-        > .iui-user-icon > img {
-          filter: $iui-icons-color-multicolor-disabled;
+      .iui-status {
+        @include themed {
+          fill: t(iui-icons-color-positive);
         }
       }
     }
-  }
+    
+    &.iui-warning {
+      &,
+      + .iui-expanded-content {
+        @include themed {
+          box-shadow: inset $iui-xxs 0 0 0 t(iui-icons-color-warning);
+        }
 
-  .iui-tables-message-container {
-    @include themed {
-      color: t(iui-text-color-muted);
-      background-color: t(iui-color-background-2);
-    }
-    display: flex;
-    justify-content: center;
-    height: 100%;
-    padding: $iui-xl;
-  }
+        *::selection {
+          @include themed {
+            background-color: rgba(
+              t(iui-color-foreground-warning-rgb),
+              t(iui-opacity-4)
+            );
+          }
+        }
+      }
 
-  .iui-tables-paginator {
-    @include themed {
-      background-color: t(iui-color-background-3);
+      .iui-status {
+        @include themed {
+          fill: t(iui-icons-color-warning);
+        }
+      }
     }
-    user-select: none;
 
-    &:hover {
-      @include themed {
-        background-color: t(iui-color-background-3) !important;
+    &.iui-negative {
```

Comment:
`.iui-positive`, `.iui-warning`, & `.iui-negative` could probably be minimized into a mixin to reduce the code since everything is identical _except_ for the color applied.

Reply:
If color is the only value changing, it could be a mixin with parameter to generate all of them.

Reply:
I just pushed an update that is using the new `iui-table-row-status` mixin.